src/adapter/src/webhook.rs

```diff
@@ -270,6 +289,26 @@ impl Default for WebhookConcurrencyLimiter {
     }
 }
 
+/// Evaluates any calls to `UnmaterializableFunc::CurrentTimestamp`, replacing it with the current
+/// time.
+fn eval_current_time(
```

Comment:
Instead of a new function, could we use `prep_scalar_expr` with a new `ExprPrepStyle`?